libdeno/modules_test.cc

```diff
@@ -266,3 +266,31 @@ TEST(ModulesTest, ImportMetaUrl) {
   EXPECT_EQ(nullptr, deno_last_exception(d));
   EXPECT_EQ(1, exec_count);
 }
+
+TEST(ModulesTest, ImportMetaMain) {
+  Deno* d = deno_new(deno_config{0, empty, empty, recv_cb});
+
+  const char* a_src = "if (!import.meta.main) throw 'hmm'";
+  static deno_mod a = deno_mod_new(d, true, "a.js", a_src);
```

Comment:
rename `a_src` to `throw_not_main_src` 
rename `a` to `throw_not_main`

rename `b_src` to `throw_main_src` 
rename `b` to `throw_main`